### PR TITLE
[Reference cleanup]LPS-162727

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang @tinatian,

This PR is to convert method references to field references in item-selector  | Misc. Thanks!
https://issues.liferay.com/browse/LPS-162727

### CODE REVIEW

**Comment by JanisZhang** [comment]
Hi @Kvale1733 Could you please help to scan the results? Thanks

**Comment by Kvale1733** [comment]
@JanisZhang No regressions found.

File: modules/apps/item-selector/item-selector-web/src/main/java/com/liferay/item/selector/web/internal/portlet/ItemSelectorPortlet.java [hunk 1]
```diff
@@ -81,18 +81,12 @@ public void render(
 		super.render(renderRequest, renderResponse);
 	}
 
-	@Reference(unbind = "-")
-	protected void setItemSelector(ItemSelector itemSelector) {
-		_itemSelector = itemSelector;
-	}
+	@Reference
+	protected ItemSelector itemSelector;
```
Comment (dantewang) [inline modules/apps/item-selector/item-selector-web/src/main/java/com/liferay/item/selector/web/internal/portlet/ItemSelectorPortlet.java:85]: @JanisZhang This portlet doesn't have sub-class, please confirm whether these referenced fields need to be `protected`.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

File: modules/apps/item-selector/item-selector-web/src/main/java/com/liferay/item/selector/web/internal/portlet/ItemSelectorPortlet.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/item-selector/item-selector-web/src/main/java/com/liferay/item/selector/web/internal/portlet/ItemSelectorPortlet.java]: @dantewang got it, thx.

**Comment by JanisZhang** [comment]
Hi @zhangyan0701, Could you please help to scan the test results? THX.

**Comment by JanisZhang** [comment]
Hi @dantewang , force-pushed, thx

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by dantewang** [comment]
@shuyangzhou @tinatian Forwarding this one as all tests passed review and changes are direct replacements.

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/123301

**Pull request closed without merging**